create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/__main__.py

```diff
@@ -2,8 +2,10 @@
 import uvloop
 {%- endif %}
 from aiohttp import web
-
```

Comment:
Why did you remove this section separator? Get it back!

Reply:
I've got it back